docs/content/users/usage/faq.md

```diff
@@ -111,13 +111,34 @@ In project A, use `mysql -h ddev-projectb-db` to access the database server of p
 
 Let’s say we have two projects, for example: project A, and project B.
 
-To enable server-side HTTP/S communication (i.e. server-side API calls) between projects you can:
+DDEV v1.24.9+ has built-in support for inter-project HTTP/S communication:
```

Comment:
No need to make a change now, but this may be getting long-ish for the FAQ, and we may want to find a new home for it and link from FAQ.